examples/nifgen_standard_function/_nifgen_helpers.py

```diff
@@ -3,10 +3,11 @@
 from typing import Any, Dict, Optional
 
 import grpc
-import ni_measurementlink_service as nims
 import nifgen
 from _constants import USE_SIMULATION
 
+import ni_measurementlink_service as nims
```

Comment:
Why is this new import required with no other changes to the file? Similar for the next two files.

Reply:
I initially reorganized these lines in response to linting errors but subsequently reverted the changes.